### ISSUE TITLE
Configuring the Administration Node

### ISSUE THREAD

**Issue by alexarnoldy** [SUSE/doc-caasp#116]
4.3  Configuring the Administration Node

https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_deployment/book_caasp_deployment.html#sec.deploy.nodes.admin_configuration

I would recommend a better description of this setting. It took me a while to figure out what this would do, and Google won't help at all in this case.

**Comment by r0ckarong** [comment]
Could you be more specific where you had trouble? The link points to an entire section with various options. Is it the overall description of this setting that should be improved? A particular option?

I'm happy to change/amend the description but I need some more information.

**Comment by alexarnoldy** [comment]
I'm so sorry for not being more clear! For some reason I must have thought that the "Report Bug" link would go straight to the screen I was on. 

I was referring to the setting: 
"Node allocation size (CIDR length per worker node)
Length of CIDR notation length per worker node in Bits (Default: 23) " 

Since the most of the online calculators don't offer this information, it was hard to know if/how this would have to change if I changed the netmask of the cluster overlay CIDR. After a little while, I finally realized that this was simply carving out a subnet of the /13 subnet for the cluster nodes (unless I'm still wrong about that!).
